pacai/util/probability.py

```diff
@@ -68,8 +64,8 @@ def sample(distribution, values = None):
 
     return values[i]
 
-def sampleFromCounter(ctr):
-    items = sorted(list(ctr.items()))
+def sampleFromDict(dict):
+    items = sorted(dict.items())
```

Comment:
Not sure what dict is. Name it something different. This is because dict is a type and naming your variable dict might confuse people.